src/docs/asciidoc/reference/manage/application-intro.adoc

```diff
@@ -0,0 +1,6 @@
+:relative-path: ../../../
+include::{docdir}/variables.adoc[]
+
+An *application* is a logical container designed to perform a group of coordinated tasks or activities for the benefit of the user.
+An application may have a front part and a worker.
```

Comment:
I think at this point of the documentation we never talked about worker. Maybe we should introduce it before or don't talk about it right now. The same applies to front. It is obvious to us but maybe not for our clients